pybaseball/amateur_draft.py

```diff
@@ -3,33 +3,42 @@
 
 from . import cache
 
+# pylint: disable=line-too-long
+_URL = "https://www.baseball-reference.com/draft/?year_ID={year}&draft_round={draft_round}&draft_type=junreg&query_type=year_round&"
 
-def get_draft_results(year, round):
-    url = f"https://www.baseball-reference.com/draft/?year_ID={year}&draft_round={round}&draft_type=junreg&query_type=year_round&"
+
+def get_draft_results(year: int, draft_round: int) -> pd.DataFrame:
+    url = _URL.format(year=year, draft_round=draft_round)
     res = requests.get(url, timeout=None).content
     draft_results = pd.read_html(res)
     return draft_results
 
+
 @cache.df_cache()
-def amateur_draft(year, round, keep_stats=True):
-    draft_results = get_draft_results(year, round)
+def amateur_draft(year: int, draft_round: int, keep_stats: bool = True) -> pd.DataFrame:
+    draft_results = get_draft_results(year, draft_round)
```

Comment:
Okay. I have bad feelings about this, because on the one hand, this absolutely shouldn't have been allowed in the first place, OTOH it is an API change. I think change it, but change the docs to match.

Reply:
We're on the EXACT same page regarding this. I didn't want to change an API parameter name, but when it's a reserved word...

Great point about the docs, I'll fix those.

Reply:
@schorrm Docs updated.